packages/x-charts/src/RadarChart/RadarDataProvider/radar.types.ts

```diff
@@ -0,0 +1,37 @@
+export interface MetricConfig {
+  /**
+   * The name of the metric.
+   */
+  name: string;
+  /**
+   * The minimal value of the domain.
+   * @default 0
+   */
+  min?: number;
+  /**
+   * The maximal value of the domain.
+   * If not provided, it gets computed to display the entire chart data.
+   */
+  max?: number;
+}
+
+export interface RadarConfig {
+  /**
+   * The different metrics shown by radar.
+   */
+  metrics: string[] | MetricConfig[];
+  /**
+   * The default max value for axis.
+   * If will be override is `metrics` contains a `max` property.
+   */
+  max?: number;
+  /**
+   * The angle of the first axis (in deg)
```

Comment:
Would it make sense to say something like "The starting angle of the rotation axis (in degrees)"? It might not be clear that the first axis is the rotation axis, or is it?